tests/e2e/default/translations/incorrect-locale.wdio-spec.js

```diff
@@ -30,12 +30,12 @@ const contact = placeFactory.place().build({
 });
 
 describe('Testing Incorrect locale', () => {
-  after(async () => {
-    await browser.setCookies({ name: 'locale', value: 'en' });
+  afterEach(async () => {
     await utils.revertSettings(true);
+    await browser.setCookies({ name: 'locale', value: 'en' });
   });
 
-  before(async () => {
+  beforeEach(async () => {
     await loginPage.cookieLogin();
     await utils.saveDoc(contact);
```

Comment:
Would this not conflict if we do it beforeEach?

Reply:
Wow yeah, this test leaves the db in a mess.

Even the `createLanguage` code will hang because although it'll silently swallow the language doc already existing, it won't updated the SW so the SW listener will never fire.

This needs a lot of work...